notebooks/openvino-tokenizers/openvino-tokenizers.ipynb

```diff
@@ -18,6 +18,7 @@
     "    - [Convert Tokenizer from HuggingFace Hub with CLI Tool](#Convert-Tokenizer-from_HuggingFace-Hub-with-CLI-Tool)\n",
```

Comment:
<p>Did you check the <code>OpenVINO/TinyLlama-1.1B-Chat-v1.0-int4-ov</code> model? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2367/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
I have checked int8 version from OpenVINO org (`OpenVINO/TinyLlama-1.1B-Chat-v1.0-int8-ov`) - output was also not fully equal ("Quick brown fox jumped over the fence." with OV vs "Quick brown fox jumped over the lazy dog." with GenAI).
So for now I returned existing model to match with `convert_tokenizer` section.

Do you have other suggestions?

Reply:
What is the default sampling config in genai?

Reply:
As far as I can see, default generation config is the following:
```
{
    "diversity_penalty": 1.0,
    "do_sample": False,
    "eos_token_id": -1,
    "frequency_penalty": 0.0,
    "ignore_eos": False,
    "length_penalty": 1.0,
    "max_length": 18446744073709551615,
    "max_new_tokens": 18446744073709551615,
    "min_new_tokens": 0,
    "no_repeat_ngram_size": 18446744073709551615,
    "num_beam_groups": 1,
    "num_beams": 1,
    "num_return_sequences": 1,
    "presence_penalty": 0.0,
    "repetition_penalty": 1.0,
    "rng_seed": 0,
    "stop_criteria": "<StopCriteria.HEURISTIC: 1>",
    "temperature": 1.0,
    "top_k": 18446744073709551615,
    "top_p": 1.0,
}
```
In GenAI `generate` method only `max_new_tokens` is set and added to the config.

If I am not mistaken, such config should relate to greedy decoding as `do_sample` is set to False and `num_beams` is set to 1

Reply:
as I know in previous time, the issue of result difference was caused by repetition_penalty (genai uses it only for beam > 1, while hf used it in gready too), possibly is it the same issue?

Reply:
I have tried to change repetition penalty (and later some other generation config params), but there was no effect..